modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets_{context}"]
+= Creating a secret using the web console
+
+You can create secrets using the web console. 
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Click *Create* -> *From YAML*.
+.. Edit the YAML manually to your specifications, or drag and drop a file into the YAML editor. 
+For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+ name: example
+ namespace: <namespace>
```

Comment:
The indentation is only one space. Inconsistent with normally even number of spaces.